docs/articles/expensify-classic/expensify-card/Auto-Reconciliation.md

```diff
@@ -35,6 +35,8 @@ To set up your auto-reconciliation account with the Expensify Card, follow these
 5. Head to the "Settings" tab.
 6. Select the account in your accounting solution that you want to use for reconciliation. Make sure this account matches the settlement business bank account.
 
+![Image showing overview of the Company Card Settings section](https://help.expensify.com/assets/images/Auto-Reconciliaton_Image1.png){:width="100%"}
```

Comment:
No need for "Image showing overview of the ". Just keep the end of the sentence. (same for line 49)

Reply:
Done- sorry for making this so painful 🙈